cdisc_rules_engine/rules_engine.py

```diff
@@ -134,9 +136,18 @@ def validate_single_rule(
                 is_split_dataset(datasets, dataset_domain),
                 datasets,
             ):
-                result: List[Union[dict, str]] = self.validate_rule(
-                    rule, dataset_path, datasets, dataset_domain
-                )
+                result = None
+                if vx in ("Y", "YES"):
```

Comment:
just use self.validate_xml here instead of introducing a new vaguely-named intermediate variable

Reply:
Used self variable name.